### PR TITLE
Enable default reference path for CopyNodeQuantizeHandler

### PR DESCRIPTION
**Pull request by terrychenism**
Summary:
In this PR we want to enable the reference path by default for CopyNodeQuantizeHandler

Test Plan:
```
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps
```

Reviewers:

Subscribers:

### CODE REVIEW

File: torch/ao/quantization/fx/quantized_fusion_patterns_and_replacements.py [hunk 1]
```diff
@@ -17,6 +17,113 @@ def relu_replacement(x, scale, zero_point):
     x = torch.nn.functional.relu(x)
     return x
 
+def relu_op_pattern(x, scale, zero_point):
```
Comment (jerryzh168) [inline torch/ao/quantization/fx/quantized_fusion_patterns_and_replacements.py:20]: nit: op -> method to align better with fx terms

**Review by jerryzh168 — APPROVED** [review]
Looks good, please also fill in the description as well, e.g. https://github.com/pytorch/pytorch/pull/69778
Summary and Test Plan are required

**Comment by jerryzh168** [comment]
you can also import the PR with `ghimport -s PR_LINK`

File: torch/ao/quantization/fx/quantization_patterns.py [hunk 2]
```diff
@@ -1487,7 +1487,14 @@ def convert(self,
...
         # always produce reference pattern for relu
-        is_relu = node.op == "call_function" and node.target == torch.nn.functional.relu
+        func_list = [
+            torch.nn.functional.relu,
+            torch.flatten,
+            torch.max,
+            torch.min,
+            torch.clamp,
+        ]
+        is_relu = node.op == "call_function" and node.target in func_list
```
Comment (jerryzh168) [inline torch/ao/quantization/fx/quantization_patterns.py:1497]: nit: need to rename `is_relu` here as well

File: torch/ao/quantization/fx/quantization_patterns.py [hunk 3]
```diff
@@ -1486,9 +1486,16 @@ def convert(self,
...
+        # always produce reference pattern for following functions
+        func_list = [
+            torch.nn.functional.relu,
+            torch.flatten,
+            torch.max,
+            torch.min,
+            torch.clamp,
+        ]
+        is_func = node.op == "call_function" and node.target in func_list
+        if is_reference or is_func:
```
Comment (jerryzh168) [inline torch/ao/quantization/fx/quantization_patterns.py:1498]: I think we want to include the two relu method as well
```
is_method = node.op == "call_method" and node.target in ["relu", "relu_"]
```

File: torch/ao/quantization/fx/quantization_patterns.py [hunk 4]
```diff
@@ -1486,9 +1486,16 @@ def convert(self,
                 load_arg: Callable,
                 is_reference: bool = False,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
-        # always produce reference pattern for relu
-        is_relu = node.op == "call_function" and node.target == torch.nn.functional.relu
-        if is_reference or is_relu:
+        # always produce reference pattern for following functions
+        func_list = [
```
Comment (jerryzh168) [inline torch/ao/quantization/fx/quantization_patterns.py:1490]: there are some missing functions here as well I think, e.g. `torch.nn.functional.relu6`, can you add all the functions supported by lowering?

**Comment by terrychenism** [comment]
same PR for this task https://github.com/pytorch/pytorch/pull/71168

**Pull request closed without merging**